src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -457,6 +474,9 @@ void Resizer::makePadParasitic(const Net* net)
 
     // Use a small resistor to keep the connectivity intact.
     parasitics_->makeResistor(parasitic, 1, .001, n1, n2);
+    if (spef_writer_) {
```

Comment:
I don't have access to the spef_files_ map outside the rsz::estimateParasitics function. I think it's simpler to not initialize the spef_writer on line 284 if the received map is empty.

Also, maybe I should change the name from spef_files_ to spef_streams_ since it will not be necessarily written to a file now that it's using std::ostream*?

Reply:
Sounds good.

Reply:
This comment wasn't addressed yet.